app/src/main/res/layout/activity_settings.xml

```diff
@@ -227,6 +227,22 @@
 
                 </RelativeLayout>
 
+                <RelativeLayout
+                    android:id="@+id/settings_hide_dialpad_numbers_holder"
+                    style="@style/SettingsHolderCheckboxStyle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@drawable/ripple_background">
+
+                    <com.simplemobiletools.commons.views.MyAppCompatCheckbox
+                        android:id="@+id/settings_hide_dialpad_numbers"
+                        style="@style/SettingsCheckboxStyle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="Hide dialpad numbers" />
```

Comment:
oh cmon, move this into the string file of all languages (in english) and fetch it from there

Reply:
will do